src/test/java/com/tom/test/excercise01/ShoppingCartTest.java

```diff
@@ -0,0 +1,55 @@
+package com.tom.test.excercise01;
+
+import com.tom.driver.manager.DriverUtils;
+import com.tom.page.object.Excercise01Page;
+import com.tom.test.TestBase;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ShoppingCartTest extends TestBase {
+
+    @Test
+    @DisplayName("Adding and removing items from cart.")
+    public void addingToCartAndRemovingFromCartItemsTest() {
+        //given
+        Excercise01Page page = new Excercise01Page();
+        int actualItemsInCart;
+        final int EXPECTED_ITEMS_IN_CART = 25;
+        BigDecimal actualPriceOfItemsInCart;
+        BigDecimal expectedPriceOfItemsInCart;
+        String pageUrl = "https://testingcup.pgs-soft.com/task_1";
+
+        //when
+        LOGGER.info("Navigating to page {}", pageUrl);
+        DriverUtils.navigateToPage(pageUrl);
+
+        //and
+        LOGGER.info("Expeted summary price calculations");
+        expectedPriceOfItemsInCart = new BigDecimal(0);
```

Comment:
use BigDecimal where needed not everwhere. BigDecimal is good for operations that require presision when dealing on floating number and for numbers that are too big for long type.
Number of items can't be floating number like 1.1 and won't be hight enought to exceed Long capacity so use int for this. BigDecinal is good for items prices